.github/workflows/merge-pr.yml

```
@@ -117,17 +117,19 @@ jobs:
           fail_ci_if_error: false # optional (default = false)
           verbose: true # optional (default = false)
 
-  save-encoded-metadata:
-    name: Save Encoded Metadata
+  save-encoded-binary-with-metadata:
+    name: Save Encoded Binary with Metadata
```

If you have to rebase, I'd swap this wording around some to be more clear.

```suggestion
  save-binary-and-encoded-metadata:
    name: Save Binary and Encoded Metadata
```